scripts/execute.py

```diff
@@ -27,9 +32,23 @@
 max_age = 100
 start_year = 2013
 pop_graphs = False
-(omega_path_S, imm_rates_path, rho_s, omega_SS, surv_rates_S, g_n_path,
-    g_n_SS, omega_S_preTP) = demog.get_pop_objs(E, S, T, min_age, max_age, start_year, pop_graphs)
-imm_rates_SS = imm_rates_path[-1, :]
+# Get the immigration rates for every period
+imm_rates_SS = demog.get_imm_resid(E+S, min_age, max_age, pop_graphs)
+
+# Get population objects
+(omega_path_S, g_n_SS, omega_SS, surv_rates_S, surv_rates,
```

Comment:
@jtregde I think you have one too many arguments coming back here.  There is only one vector of survival rates for each age.

You can test this by trying to execute this script and seeing if you get an error here.

Reply:
Ah, I see, I had that extra `surv_rates` in there, but was missing that it returns the `imm_rates_mat` as well. It appears there are 8 things being returned from the `get_pop_objs` function, right?

Reply:
@jtregde You are right - 8 objects returned.